### ISSUE TITLE
Reflex Sight modding example no longer works

### ISSUE THREAD

**Issue by NotAFile** [yvt/openspades#632]
https://github.com/yvt/openspades/wiki/Example%3A-Reflex-Sight

It tries to search for files called `SMG/shotn.wav`, which seem to no longer exist in current OpenSpades.

**Comment by NotAFile** [comment]
So uh I gave it a try and here is a new version that works:

https://gist.github.com/NotAFile/dc31929e88c58632ed6d24f75b5f84e1

**Comment by yvt** [comment]
Looks like some user-created weapon MODs are affected by this as well. I guess I should distribute a "compatibility pak" that provides the `.wav` version of sound files.

**Comment by KapitanUkraina** [comment]
@yvt yeah i can confirm that this method works, i am now using my old weapons and they're working. Some stuff is still broken tho like weapon sounds because its not using any "distance" sounds propably because they're not set in script by default anywhere. I will try to add them manually but most important part - old weapon skins are working! yay!

**Comment by BR-** [comment]
It would be nice to have this built in as a setting, instead of a mod.

**Comment by NotAFile** [comment]
I'd disagree, it would probably be more useful to be able to switch mods with a real menu and then ship a few options with OpenSpades.
